src/kernel/include/shared/cos_types.h

```diff
@@ -496,7 +497,8 @@ enum {
 
 enum {
 	MAPPING_READ  = 0,
-	MAPPING_RW    = 1
+	MAPPING_RW    = 1,
+	MAPPING_NO_VALLOC = 2
```

Comment:
this feels like a very strange place to have this.  I may be off here, but it has to do with a specific component in a file that has nothing to do with specific components.  I'd say there is likely a 99% chance this should not be here.